ONSdigital/census-rh-ui
# Cr 119 refactor rh ui to use rhsvc

# Motivation and Context
The RH UI has been changed to use the RHSVC  instead of individual RM services and a new EQ payload has been created.

# What has changed
All calls to individual RM services have been removed and a single call is made to the RHSVC which returns all information in a json in one go.
The EQ payload has changed and for the purposes of progress some values have been hard-coded until decisions have been made ( region, language ) and some are hardcoded in as they will not be needed in the new payload eventually but until EQ has made the changes then they need to be kept in.


# How to test?
Local integration testing with the RHSVC can be done by running both locally and 'mimicking' putting things on the queues as they would from RM.
Unit tests that are no longer required have been removed.
Unit tests have been refactored to make use of the RHSVC instead of individual RM services and to make the new EQ payload.


# Links
JIRA cards CR-119, CR-159 and Cr-204 have all been delivered as part of this work.

# Screenshots (if appropriate):
